06-wt-multilevel-models-1.Rmd

```diff
@@ -78,7 +78,7 @@ datasets. We assign each of the datasets a name using `<-`.
 ``` {r}
 # Gradebook and log-trace data for F15 and S16 semesters
 s12_course_data <-
-  read_csv(here(
+  read_csv(here::here(
```

Comment:
thanks for adding this - do you think this is necessary even though we have the here package loaded?

Reply:
It's caused bugs for me in the past so I've been using `here::here()`. I can't remember which package it conflicts with though.

Reply:
kk, keeping here::here()

Reply:
ok, this was a case for which we ended up loading data from the package, so is simplified/a bit clearer now (just `course_data <- dataedu::course_data)`).